e2e/testdata/fn-render/fn-success-with-stderr/.expected/results.yaml

```diff
@@ -5,6 +5,5 @@ metadata:
 exitCode: 0
 items:
   - image: gcr.io/kpt-fn/starlark:v0.2
-    stderr: |
-      function succeeded, reporting it on stderr
+    stderr: function succeeded, reporting it on stderr
```

Comment:
Is this change necessary?

Reply:
This is needed because we are stripping away `\n` from the last line returned by the function when sanitizing the output from the function. It's because we are using `bufio.Scanner` to process the output. I might be able to work around it, but didn't think it matters. Do you think of it being a problem ?

Reply:
I don't think that's a problem.